docs/book/src/topics/managedcluster.md

```diff
@@ -277,6 +277,31 @@ User can do that by adding the Service Principal to the appropriate group define
 add the corresponding group ID in `spec.aadProfile.adminGroupObjectIDs`. 
 CAPI and CAPZ will be able to authenticate via AAD while accessing the target cluster.
 
+### AKS Fleet Integration
+
+CAPZ supports joining your managed AKS clusters to a single AKS fleet. Azure Kubernetes Fleet Manager (Fleet) enables at-scale management of multiple Azure Kubernetes Service (AKS) clusters. For more documentation on Azure Kubernetes Fleet Manager, refer [AKS Docs](https://learn.microsoft.com/azure/kubernetes-fleet/overview)
+
+To join a CAPZ cluster to an AKS fleet, you must first create an AKS fleet manager. For more information on how to create an AKS fleet manager, refer [AKS Docs](https://learn.microsoft.com/en-us/azure/kubernetes-fleet/quickstart-create-fleet-and-members). This fleet manager will be your point of reference for managing any CAPZ clusters that you join to the fleet.
```

Comment:
Just to clarify, the AKS fleet manager is something that cannot be created with CAPZ and lives independently from CAPZ, right?

Reply:
Yes it is something the user will create outside of CAPZ before spinning up a cluster with a fleets member